docs/source/markdown/podman-kube-generate.1.md

```diff
@@ -34,6 +34,11 @@ Note that the generated Kubernetes YAML file can be used to re-run the deploymen
 
 Output to the given file instead of STDOUT. If the file already exists, `kube generate` refuses to replace it and returns an error.
 
+#### **--no-trunc**
+
+Don't truncate annotations to Kubernetes length of 63 characters
+Note: enabling this flag means the generated YAML file is not Kubernetes compatible
```

Comment:
Maybe add "and can only be used with `podman play kube`?

Reply:
`podman kube play` also restricts the user to 63 character long annotations. Anything longer will result in the command returning an error

Reply:
We'll either want to drop that to a warning, or add a flag to allow overlong annotations there - the intent here was to pair the files generated here with `podman play kube` to run them

Reply:
done